lamindb/core/writelog/_trigger_installer.py

```diff
@@ -632,9 +632,9 @@ def install_triggers(self, table: str, cursor: CursorWrapper):
 
 def create_writelog_recording_trigger_installer(
     connection: BaseDatabaseWrapper,
-) -> WriteLogRecordingTriggerInstaller:
+) -> WriteLogDBRecordingTriggerInstaller:
```

Comment:
Looks like an incidental rename to me.

Reply:
Thanks for spotting! I wasn't yet through with reviewing but this is a great catch!